components/wpt-results.html

```diff
@@ -138,12 +138,20 @@
           testRuns: {
             type: Array,
           },
+          filteredDirectories: {
+            type: Array,
+            value: []
```

Comment:
Answer for 1&2: yes correct, the reason it's an array is to allow (in the future) spec or API owners (or anyone else) to set up custom views that feature a few different directories, like:

`wpt.fyi/?dirs=css/CSS2/borders,css/CSS2/normal-padding`